src/libmongoc/src/mongoc/mongoc-crypt.c

```diff
@@ -628,6 +630,32 @@ _needs_on_demand_aws_kms (bson_t const *kms_providers, bson_error_t *error)
    }
 }
 
+/**
+ * @brief Check whether the given kmsProviders object requests automatic Azure
+ * credentials, and what the managedIdentity parameters might be
+ *
+ * @param kmsprov The input kmsProviders that may have an "azure" property
+ * @param error An output error
+ * @return true If success AND `kmsprov` requests automatic Azure credentials
+ * @return false Otherwise. Check error->code for failure.
```

Comment:
```suggestion
 * @return true on success and `kmsprov` requests automatic Azure credentials.
 * False otherwise. Check error->code for failure.
```

Reply:
This documentation shape was stamped out by Doxygen generator I use, but it looks to be goofing here. It _should_ be using the `@retval` command, which consumes the following token as the return value, and the remaining line as a description corresponding to that value.